mesonbuild/coredata.py

```diff
@@ -258,6 +259,9 @@ def listify(self, value: T.Union[str, T.List[str]]) -> T.List[str]:
             raise MesonException(f'"{value}" should be a string array, but it is not')
         return newvalue
 
+    def listify(self, value: T.Any) -> T.List[T.Any]:
```

Comment:
Any reason to keep the non-static one around?

Reply:
Yes, it overrides the base class implementation. We do `opt.listify()` in `coredata.set_option()`. The reason I add a staticmethod version here is to be able to reuse that code elsewhere: It's used in mcompile and the next commit uses it in UserStdOption.

Reply:
Ok. Thanks for the explanation. I wonder if there is a better name for the static function, but as-is is also fine.